p2p/antiflood/topicFloodPreventer.go

```diff
@@ -0,0 +1,79 @@
+package antiflood
```

Comment:
can we reuse the elrond-go/process/throttle/antiflood/topicFloodPreventer's code?